agent/handlers/taskmetadata/helpers.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Can you move this file under handlers?

Reply:
ok, but why?

Reply:
I think the method can be used by other handler too, not specific to task metadata handler. like `writeJSONToResponse`

Reply:
We'll do that refactor when it's actually needed. For now, i'll keep it this way to limit the number of changes. Else, a bunch of non exported fields will now have to be exported and I want to avoid that churn in the interest of time.